smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -403,7 +382,7 @@ interface IPublicLock
   ///===================================================================
   /// Auto-generated getter functions from public state variables
 
-  function beneficiary() external view returns (address );
+  function beneficiary() external view returns (address);
```

Comment:
Thx should go, right?

Reply:
true. I marked it as private so it wont appear anymore - even though we keep it for storage layout

Reply:
Great!